en/flight_modes/offboard.md

```diff
@@ -61,21 +61,26 @@ bool actuator
 ```
 
 The fields are ordered in terms of priority such that `position` takes precedence over `velocity` and later fields, `velocity` takes precedence over `acceleration`, and so on.
-The first field that has a non-zero value (from top to bottom) defines what valid estimate is required in order to use offboard mode, and the setpoint message(s) that can be used.
-For example, if the `acceleration` field is the first non-zero value, then PX4 requires a valid `velocity estimate`, and the setpoint must be specified using the `TrajectorySetpoint` message.
+The first field that has a non-zero value (from top to bottom) defines what valid estimate is required in order to use offboard mode, the setpoint message(s) that can be used, and the controllers that are enabled.
+For example, if the `acceleration` field is the first non-zero value, then PX4 requires a valid `velocity estimate`, the setpoint must be specified using the `TrajectorySetpoint` message `acceleration` field, and the position and velocity controllers are disabled (so that even if set, the setpoint position and velocity field will not be used).

```

Comment:
The statement about what controllers are disabled is quite significant. I thought worth expanding out here below in the example.

Reply:
My apologies, this

> the position and velocity controllers are disabled (so that even if set, the setpoint position and velocity field will not be used).

 is not correct, see https://github.com/PX4/PX4-user_guide/pull/2341#issuecomment-1469377993
 
- It needs just one of `position`, `velocity`, `acceleration` fields to be set to enable all controllers. Users need to use appropriately the setponint fields (set them to `NaN`) to have additional control.
- `attitude` set enables attitude controller and rate controller.
- `body_rate` set enables only the rate controller.